beetmoverscript/script.py

```diff
@@ -82,6 +89,27 @@
     await retry_download(context=context, url=source, path=abs_file_path)
     await retry_upload(context=context, destinations=destinations, path=abs_file_path)
 
+    if context.task['payload']['update_manifest']:
+        enrich_balrog_manifest(context, abs_file_path, locale)
```

Comment:
actually, since we only care about the multi apk (we ignore the en-us one from the signed task), we might have to tweak this part now.

We also have the added bonus that the `locale` is a lie since we tell balrog to put the 'multi' apk under 'en-us' locale